src/test/java/sorinfoca/tests/MyTests.java

```diff
@@ -0,0 +1,90 @@
+package sorinfoca.tests;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import sorinfoca.driver.BrowserManager;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class MyTests extends TestConfig {
+
+    WebDriver driver = new ChromeDriver();
+
+    @BeforeEach
```

Comment:
Use @BeforeMethod from TestNG library and move it to TestConfig as it should be in a separated class inherited by test class(es)